### PR TITLE
Code review and accessibility

### PR DESCRIPTION
**Pull request by Sybrand**
- Addressed issue from code review.
- Made some change to improve accessibility score.

### CODE REVIEW

**Comment by Sybrand** [comment]
[EPIC#Registry | Add new applicant - Classification and Qualifications |5| !!! [gherkin]](https://trello.com/c/5xbTW0Fz/795-epicregistry-add-new-applicant-classification-and-qualifications-5-gherkin)

**Comment by Sybrand** [comment]
[EPIC#Registry | Add new applicant - Adjudication |0| !!! [gherkin]](https://trello.com/c/vYd6OvhV/800-epicregistry-add-new-applicant-adjudication-0-gherkin)

File: app/frontend/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```
Comment (stephenhillier) [inline app/frontend/index.html]: No big deal but I don't think we are using this file at the moment

Comment (Sybrand) [reply app/frontend/index.html]: It was bugging me during accessibility checks on my localhost - big red flag if it's not specified.

**Pull request merged**